src/compiler/program.ts

```diff
@@ -4914,6 +4933,17 @@ export function createProgram(rootNamesOrOptions: readonly string[] | CreateProg
         return _compilerOptionsObjectLiteralSyntax || undefined;
     }
 
+    function getCompilerOptionsKeySyntax() {

```

Comment:
Can this method (so cached result) be used in other methods that get option of compiler Options  `getCompilerOptionsObjectLiteralSyntax`